clientv3/retry.go

```diff
@@ -75,12 +71,8 @@ func (c *Client) newRetryWrapper(isStop retryStopErrFunc) retryRPCFunc {
 			if isStop(err) {
 				return err
 			}
-			select {
-			case <-notify:
-			case <-rpcCtx.Done():
-				return rpcCtx.Err()
-			case <-c.ctx.Done():
-				return c.ctx.Err()
+			if err = readyWait(rpcCtx, c.ctx, notify); err != nil {
```

Comment:
is this really necessary? the for loop will do the readywait again at the top.

Reply:
Just removed.
In retry fix PR, we should move the one on the top (no pinned address, then it will block forever in our integration tests).